### ISSUE TITLE
'module' object has no attribute 'Ward'

### ISSUE THREAD

**Issue by dodo5575** [msmbuilder/msmbuilder#692]
After installing msmbuilder: 3.3.0-np110py27_1 with conda, I tried to run examples on http://msmbuilder.org. But I got some error while just loading the modules. 

Here is the code:

```python
%matplotlib inline
import numpy as np
from matplotlib import pyplot as plt
from mdtraj.utils import timing
from msmbuilder.example_datasets import load_doublewell
from msmbuilder.cluster import NDGrid
from msmbuilder.msm import BayesianMarkovStateModel, MarkovStateModel
```

Here is the error message:

```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-1-b40dc487311c> in <module>()
      4 from mdtraj.utils import timing
      5 from msmbuilder.example_datasets import load_doublewell
----> 6 from msmbuilder.cluster import NDGrid
      7 from msmbuilder.msm import BayesianMarkovStateModel, MarkovStateModel

/Applications/anaconda/lib/python2.7/site-packages/msmbuilder/cluster/__init__.py in <module>()
     76 
     77 
---> 78 class Ward(MultiSequenceClusterMixin, cluster.Ward, BaseEstimator):
     79     __doc__ = _replace_labels(cluster.Ward.__doc__)
     80 

AttributeError: 'module' object has no attribute 'Ward'
```

Can anyone help? Thanks.

**Comment by rmcgibbo** [comment]
There's an incompatibility with the latest release of scikit-learn. You can upgrade to the latest conda build of msmbuilder, `3.3.0-np110py27_2`, or `conda install sklearn==0.16`.

**Comment by mpharrigan** [comment]
I thought we changed the conda recipe to force <0.17

**Comment by mpharrigan** [comment]
cc #685

**Comment by mpharrigan** [comment]
Ok, we released version 3.3.1 that fixes the root cause of this issue
